### DISCUSSION TITLE
Help needed: fd and zoxide not work with yazi on ubuntu

### DISCUSSION THREAD

**Discussion by gapplef** [sxyazi/yazi#487] [1. Q&A] [answered]
Ubuntu 22.04, Yazi v0.1.5
I download the binary from the Github release page, and installed `fd-find`, `ripgrep`, `zoxide`, `fzf`.

The `S`, `Z` key seems work, but `s`, `z` doesn't work.
- press `s` can bring up the search interface, but show nothing after search.
- press `z` have no response at all, and after quit yazi I got the following error:
  `error: Found argument '--exclude' which wasn't expected, or isn't valid in this context`

Anyone can help?

**Comment by sxyazi** [comment]
What's your `fd` and `zoxide` version?

**Reply by gapplef** [reply]
fd-find [8.3.1](https://packages.ubuntu.com/jammy/fd-find)、zoxide [0.4.3-2](https://packages.ubuntu.com/jammy/zoxide)

**Reply by sxyazi** [reply, marked as answer]
It's too old; it's been 4 years since then. Please upgrade to the latest version.
